pkg/kube/kubehttp.go

```diff
@@ -133,12 +134,18 @@ func (k *KubeHTTP) GetClusterStatus(ctx context.Context) ClusterStatus {
 		Namespace: "kube-system",
 	}
 
+	notFoundErr := `deployments.apps "coredns" not found`
+
 	if err := k.Client.Get(ctx, coreDnsName, &dep); err != nil {
-		// Couldn't find the coredns deployment.
-		// We don't know what state the cluster is in.
+		// Some clusters don't have 'coredns'; if we get a "not found" error, we know we
+		// can talk to the cluster
+		if strings.Contains(err.Error(), notFoundErr) {
```

Comment:
Might check out `apierrors.IsNotFound()`

Reply:
Thanks! Was not aware of that.

Reply:
Doh! In the same file.